platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/EventCounter.java

```diff
@@ -50,4 +60,9 @@ public static void eventCreated() {
     public static void eventCleared() {
         numEventsInMemory.decrementAndGet();
     }
+
+    public static void registerMetrics(@NonNull final Metrics metrics) {
```

Comment:
Consider renaming method to match the pattern of other new metric collection classes `registerEventCounterMetrics`

Reply:
renamed